src/wallet/wallet.h

```diff
@@ -974,6 +974,14 @@ class SpendableInputs {
         const CAmount dustThreshold,
         const std::set<libzcash::OutputPool>& recipientPools);
 
+
+    size_t GetTotalSize() const {
```

Comment:
This doesn't look like it's used in this PR? Also this just measures the count of the inputs, so it should be named differently if it's intended to be used.

Reply:
Yep, left over from the branch I originally pulled this from. Gone now.